src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -120,7 +120,7 @@ static bool queryForward(ConnectionInfo* ciP, QueryContextRequest* qcrP, QueryCo
   std::string     out;
```

Comment:
As far as I understand, no change was neded in the query forwarding logic. I'm a bit surprised (although happy :) about it has worked without changing anythign. That's make me think that the query done by mongoBackend to recover registrations takes into account registrations that can use type `".*"` (by change? by wisdom when we designed it in the past?).

Reply:
Well, only very simple cases work, if you complicate things just a little, it stops working.
The whole things needs a complete rewrite.
Seems like we misunderstood the problem. Probably we didn't think enough and as it worked for simple cases we thought everything was perfect.

That said, the only thing that needed to be fixed for the easy cases to work was to fix the registration creation part (that threw away the idPattern without saying anything).

Reply:
NTC